### PR TITLE
user audit log crated on colum fix

### PR DESCRIPTION
**Pull request by harishshg**
Fixes # .

Changes proposed in this pull request:
- fixed created_on date column fix 

@deepfence/engineering

### CODE REVIEW

File: deepfence_frontend/apps/dashboard/src/utils/date.ts [hunk 1]
```diff
@@ -11,3 +11,7 @@ export const formatMilliseconds = (date: number | string) => {
 export const formatToRelativeTimeFromNow = (date: string | Date | number) => {
   return dayjs(date).fromNow();
 };
+
+export const formatToString = (date: string | Date | number) => {
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/utils/date.ts:15]: use `formatMilliseconds`, we don't need this function. add date type to `formatMilliseconds` if needed

File: deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx [hunk 2]
```diff
@@ -119,7 +119,7 @@ const UserAuditLogs = () => {
                       data={logs}
                       columns={columns}
                       enablePagination
-                      pageSize={30}
+                      pageSize={10}
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx:122]: keep page size 30. check column widths to make sure table looks good on 720p

**Review by manV — APPROVED** [review]

File: deepfence_frontend/apps/dashboard/src/utils/date.ts (hunk 1 above)
Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/utils/date.ts]: updated formatMilliseconds to accept date

File: deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx (hunk 2 above)
Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx]: kept page size 30.

Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx]: for 720px showing 7 columns is like squished with less real estate but all the columns looks find with ellipsis wherever applicable with current width values

**Pull request merged**